app/index.js

```diff
@@ -0,0 +1,24 @@
+var OSinfo = require('../modules/OSinfo');
+
+process.stdin.setEncoding('utf-8');
+process.stdin.on('readable', function(){
+    var input = process.stdin.read();
+    if(input !== null){
+        var instruction = input.trim();
+        switch(instruction){
```

Comment:
1) add space after `switch`

2) less nesting is better, so try to move your switch outside `if` statement